api/envoy/config/cluster/aggregate/cluster.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package envoy.config.cluster.aggregate;
+
+option java_outer_classname = "ClusterProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.cluster.aggregate";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Aggregate cluster configuration]
+
+// Configuration for the aggregate cluster. See the :ref:`architecture overview
+// <arch_overview_aggregate_cluster>` for more information.
+message ClusterConfig {
+  // Load balancing clusters in aggregate cluster with explicit priority.
+  message LbCluster {
+    // Cluster name.
+    string name = 1;
+
+    // [#not-implemented-hide:] Cluster priority.
```

Comment:
Can you add some comments on how priority works in the absence of this field?

Reply:
Done